src/whatsnew/2.2.rst

```diff
@@ -27,30 +27,283 @@ Upgrade Notes
 
 .. rst-class:: open
 
-* Item 1
+* The minimum supported version of Erlang is now R17, not 16B03. Support for Erlang 21
+  is still ongoing and will be provided in a future release.
 
-* Item 2
+* TODO WORDING: Recommend increasing PBKDF2 / bcrypt work factor and auth_plugins to
+  couch_replicator_auth_session,couch_replicator_auth_noop. Do not do this, if an older 
```

Comment:
There is a PR to make `couch_replicator_auth_session,couch_replicator_auth_noop` the default. 

https://github.com/apache/couchdb/pull/1462

If that goes into 2.2 we can omit the part about this config.

 Also we'd want to remove bcrypt reference as it was backed out due to Windows build failures.

Reply:
The PR has landed in 2.2 and I have rewritten the section.